scripts/install_vim.sh

```diff
@@ -1,88 +1,186 @@
 #!/bin/bash
 
 set -e
+set -x
```

Comment:
I think it's quite nice in general, although too verbose with `echo`s for sure.
We could have a `VERBOSE` var for this, but I think the default should be 1 then maybe after all: it shows up nicely in the output in red lines, so you see what is being executed currently.